Gemfile

```diff
@@ -66,7 +66,7 @@ gem "active_model_serializers", "0.8.4"
 gem 'activerecord-session_store'
 gem 'acts-as-taggable-on', '~> 8.0'
 gem 'angularjs-file-upload-rails', '~> 2.4.1'
-gem 'bigdecimal', '1.4.2' # There are issues using later versions of bigdecimal
+gem 'bigdecimal', '3.0.2'
```

Comment:
better remove the version all together? or leave it as ~> 3.0?

Reply:
I guess dependabot will bump it anyway whenever possible?

I'm not sure about version pinning for `BigDecimal` :man_shrugging:

It looks like 3.0.2 is the very latest and it's designed for recent Ruby versions and compatibility with Ruby 3.0